yarn.lock

```diff
@@ -5,41 +5,49 @@
 "@articulate/spy@^0.0.1":
   version "0.0.1"
   resolved "https://registry.yarnpkg.com/@articulate/spy/-/spy-0.0.1.tgz#2da9fdb919075fa5fc3868017e17b43b8528a399"
+  integrity sha512-Ewt95Ba4Kg/n28OMqm/bfUOeq7vURjYL1EXmPOVzqMPPG1tBGPcEftQzlcIb+cz9rNh/5d7uxdPo9eqD8x5hXg==
```

Comment:
Why did yarn.lock thrash? Where did these come from?

Reply:
The `yarn` version we use in our :docker: containers is `1.10.1`, and although lastest is `1.12.x`, I'd recommend not bumping to that because it breaks `nyc`.  I think starting in `1.10.x`, `yarn` now adds integrity sha entries, and auto-updates the `yarn.lock` when possible.